generate-audio.py

```diff
@@ -61,6 +67,33 @@ def load_captions(config):
         raise Exception('Unsupported subtitles format')
 
 
+# TODO figure out better way of defining break length
+def define_break(diff_length, num_of_pauses):
```

Comment:
To jest jedyny fragment z którego zrozumieniem mam problem.

Jeśli mamy 2 sek różnicy (diff_length=2.0) oraz 3 przecinki (num_of_pauses=3) to domyślna długość pauzy wyniesie 0,66(6) sekundy.

Potem jest mapowanie:
Jeśli domyślna długośc pauzy jest za dluga (powyżej 2 sek) to przytnij do 1 sek
Jeśli domyślna długośc pauzy jest między 1 a 2 sek to przytnij do 0,8 sek
Jeśli domyślna długośc pauzy jest poniżej 1 sek to przytnij do 0,5 sek
 
Dlaczego jest potrzebne to mapowanie? Dlaczego nie można po prostu zwrócić length_of_pause?